eslint.config.mjs

```diff
@@ -92,10 +94,6 @@ export default tseslint.config([
       'prefer-const': ['error', { destructuring: 'all' }],
       'no-duplicate-imports': 'error',
 
-      // Blocks double-quote strings (unless a single quote is present in the
-      // string) and backticks (unless there is a tag or substitution in place).
-      quotes: ['error', 'single', { avoidEscape: true }],
```

Comment:
This was turned into the `@stylistic/quotes` rule.

Reply:
For posterity: it looks like `quotes` was deprecated and the official recommendation is to use `@stylistic/quotes` instead.